### PR TITLE
Fix aws func integration tests

### PR DESCRIPTION
**Pull request by juanjoman**
Fixing the integration tests that failed the other day. DynamoDB scans were returning +1MB of data so the events response was incomplete

Also, it seems that aws-load tests were failing

### CODE REVIEW

File: packages/framework-provider-aws-infrastructure/src/test-helper/aws-counters.ts [hunk 1]
```diff
@@ -14,24 +15,45 @@ export class AWSCounters {
   }
 
   public async events(): Promise<number> {
-    // TODO: This way of counting is wrong when there are a lot of events in the event store, as COUNT will only contain
-    // the number of elements in a a result set that is lower than 1MB in size:
-    // "If the size of the Query result set is larger than 1 MB, then ScannedCount and Count
-    // will represent only a partial count of the total items.
-    // You will need to perform multiple Query operations in order to retrieve all of the results\"."
```
Comment (juanjoman) [inline packages/framework-provider-aws-infrastructure/src/test-helper/aws-counters.ts:21]: This seemed to be the issue

**Comment by juanjoman** [comment]
checking load tests, they were working before 🤔

**Review by javiertoledo — APPROVED** [review]
LGTM!

**Comment by NickSeagull** [comment]
Load tests aren't successful yet, making it a draft

**Comment by juanjoman** [comment]
### Pasting here the message I shared on Discord:

While researching all of this, it's seems that the serverless-artillery repo dissapeared: https://www.reddit.com/r/serverless/comments/otdmqz/serverless_artillery_github/

> The repo gives a 404: https://github.com/Nordstrom/serverless-artillery

Apart from that, while researching more and more, I found out that serverless-artillery deploys a lambda with node10.x, which is going to be soon deprecated by AWS (August 30th is the end of phase 2): https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html?icmpid=docs_lambda_console.

So I'm leaving it as it was, just by fixing the aws func tests and skipping the load tests until we find a new tool to do this

**Review by NickSeagull — APPROVED** [review]

**Pull request merged**